### PR TITLE
Explicit library linking

### PR DESCRIPTION
**Pull request by philliptee**
Add two cmake macros that explicitly link against openexr and pthread libraries.
This change enables compiling and linking on the ubuntu linux distributions

### CODE REVIEW

**Comment by dictoon** [comment]
Thanks! Glad that you found the problem!

File: src/bcd/CMakeLists.txt [hunk 1]
```diff
@@ -61,6 +61,15 @@ add_library (bcd STATIC ${sources})
 apply_static_lib_linker_flags (bcd)
 
 
+#--------------------------------------------------------------------------------------------------
+# Static libraries.
+#--------------------------------------------------------------------------------------------------
+
+link_against_platform (bcd)
+
```
Comment (dictoon) [inline src/bcd/CMakeLists.txt]: You can remove the blank line here.

Comment (philliptee) [reply src/bcd/CMakeLists.txt]: Are you sure about this? The spacing I used in this commit matches the style of this file and other files where I see these macros used.

Comment (dictoon) [reply src/bcd/CMakeLists.txt]: Your code is perfectly formatted, very much appreciated. I was just suggesting grouping the two linking commands but it's such a minute detail, forget about it :)

**Comment by philliptee** [comment]
Commit formatted and rebased properly! :)

**Comment by dictoon** [comment]
Perfect, thank you!

**Pull request merged**